Dockerfile

```diff
@@ -30,3 +38,6 @@ ENV LD_LIBRARY_PATH $HADOOP_PREFIX/lib/native
 #RUN yum -y install R
 
 ENTRYPOINT ["/etc/bootstrap.sh"]
+
+HEALTHCHECK --interval=30s --timeout=5s --retries=3 \
+ CMD $SPARK_HOME/yarn-remote-client/yarn_is_ready.sh
```

Comment:
For consistency, indentation is 2 spaces for unfinished lines. Same above lines 7 to 10.